src/components/Modals/Settings/SettingsCommon.ts

```diff
@@ -7,7 +8,12 @@ export enum SettingsRoutes {
   CurrencyFormat = '/settings/currency-format',
 }
 
-export const currencyFormatsRepresenter: Record<CurrencyFormats, string> = {
-  [CurrencyFormats.DotDecimal]: '1,234.56',
-  [CurrencyFormats.CommaDecimal]: '1 234,56',
+export const currencyFormatsRepresenter = (
+  format: CurrencyFormats,
+  currencySymbol: SupportedFiatCurrencies,
+  value: number = 123456.78,
```

Comment:
This isn't actually programmatic and is never passed as a parameter, so we're kind of lying on the arity here. Can we removed the third arg and keep this as a string literal on l.17? 🙏🏽